src/sentry/snuba/subscriptions.py

```diff
@@ -187,7 +190,10 @@ def delete_snuba_subscription(subscription):
         # TODO: Move this call to snuba into a task. This lets us successfully delete a
         # subscription in postgres and rollback as needed without having to create/delete
         # from snuba
-        _delete_from_snuba(subscription)
+        try:
+            _delete_from_snuba(QueryDatasets(subscription.dataset), subscription.subscription_id)
+        except Exception:
+            logger.exception("Failed to delete unused subscription from snuba.")
```

Comment:
Would you want the subscription ID and dataset in this log line?

Reply:
Dang, I put this try/catch in the completely wrong place. Wanted to make sure failures wouldn't block the consumer